### PR TITLE
convert code blocks to md

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 1]
```diff
@@ -71,16 +71,14 @@ Use title case for headings.
...
+```ReST
+- Block one
+- Block two
+- Block three
+  - Sub 1
+  - Sub 2
+- Block four
+```
 
 ## Code blocks
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:83]: This should have an example of syntax

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 2]
```diff
@@ -115,40 +112,32 @@ Make sure to link to the raw version of the file on GitHub. There is a handy ``R
 ## Tabs
 To create code tabs in an article, you can use the ``.. tab-set-code::`` directive.  You can use ``code-block`` and ``rli`` directives inside. The format is:
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:113]: This should be updated to not referenced code-block

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 3]
```diff
@@ -115,40 +112,32 @@ Make sure to link to the raw version of the file on GitHub. There is a handy ``R
...
-         // Send our generated trajectory to Field2d.
-         m_field.GetObject("traj")->SetTrajectory(m_trajectory);
+```ReST
+.. tab-set-code::
+   .. rli:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2024.3.2/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/ramsetecontroller/Robot.java
+      :language: java
+      :lines: 44-61
+      :linenos:
+      :lineno-start: 44
+   .. code-block:: c++
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:122]: This should be updated

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 4]
```diff
@@ -299,19 +287,14 @@ The category ``index.rst`` file can then be added to the root index file located
...
-
-      System.out.println("Hello World");
-
-   ## Section
-
-   This is a section!
+```ReST
+# Title
+This is an example article
+.. code-block:: java
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:293]: This should be updated

**Comment by TheTripleV** [comment]
It never crossed my mind that I would have to support nested code blocks lol

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 5]
```diff
@@ -71,21 +71,27 @@ Use title case for headings.
...
+  - Sub 1
+  - Sub 2
+- Block four
+```
 
 ## Code blocks
 
 All code blocks should have a language specified.
 
+````md
```
Comment (jasondaming) [inline source/docs/contributing/frc-docs/style-guide.rst:87]: It doesn't look like this is working I don't see a code block

Comment (TheTripleV) [reply source/docs/contributing/frc-docs/style-guide.rst:87]: image

seems to be fine?

**Review by jasondaming — APPROVED** [review]

**Pull request merged**